cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -430,31 +433,27 @@ func ValidateIgnorePreflightErrors(ignorePreflightErrors []string, skipPreflight
 }
 
 // ValidateKubeletConfiguration validates kubelet configuration and collects all encountered errors
-func ValidateKubeletConfiguration(c *kubeadm.KubeletConfiguration, fldPath *field.Path) field.ErrorList {
+func ValidateKubeletConfiguration(c *kubeletconfigv1beta1.KubeletConfiguration, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-
-	if c.BaseConfig == nil {
+	if c == nil {
 		return allErrs
 	}
 
 	scheme, _, err := kubeletscheme.NewSchemeAndCodecs()
 	if err != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath, "kubeletConfiguration", err.Error()))
+		allErrs = append(allErrs, field.Invalid(fldPath, "", err.Error()))
```

Comment:
Why removing the field name? I think that it useful for getting more detailed error messages

Reply:
This is not the field name, fldPath gives us that. The second param is the actual value, which is unnecessary to give for a full struct. More detail exists in the error message already anyway.